.github/workflows/example-builds.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.0.5', '1.2', '^1.5.0-beta1', '1']
+        julia-version: ['1.0.5', '1.2', '^1.5.0-beta1', '1', 'lts']
```

Should we also test `pre`, since that's added in this PR?

```suggestion
        julia-version: ['1.0.5', '1.2', '^1.5.0-beta1', '1', 'lts', 'pre']
```